docs/2101UserGuide.md

```diff
@@ -62,50 +62,56 @@ CCACommander Ultra Promax Xtra 9000PLUS is the one-stop app for CCA Heads to man
 
 <hr class="feature-class-separator">
 
-### Member commands
+### Member Commands
+The vitality of a CCA lies in its members. CCACommander allows you to manage your members with the commands below:
 
 #### Create a Member: `createMember`
-Creates a new member with accompanying personal details (name, gender, phone number, email address, home address, tag).
+When you first start CCACommander and want to populate CCACommander with your members, you can do it with just one simple command - `createMember`!
 
-Format: `createMember n/MEMBER_NAME g/GENDER [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Here is how you can use it:<br>
+`createMember n/MEMBER_NAME g/GENDER [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
 
-* Acceptable values for `GENDER`: `Male`, `Female`, `Others`.
-* Acceptable values for `EMAIL`: A string with an email extension (e.g. `@gmail.com`).
+Here is an example command where we create a member **named** "Lim Jun Jie" with **gender** "Male", **phone number** of "83498018", **email** of "limjunjie@gmail.com", home **address** of "19 Kent Ridge Crescent, Singapore 119278" and a "publicity" **tag**:
 
-Examples:
-* `createMember n/CHU WEI RONG g/Male p/98765432 e/chuweirongrocks@gmail.com a/19 Kent Ridge Crescent, Singapore 119278 t/Leader` creates a member `CHU WEI RONG` in CCACommander.
+`createMember n/Lim Jun Jie g/Male p/83498018 e/limjunjie@gmail.com a/19 Kent Ridge Crescent, Singapore 119278 t/publicity`
+
+<figure>
+    <img src="images/createMemberPostCommand.png"
+         alt="createMember Post Command">
+    <figcaption>After executing the <code>createMember</code> command</figcaption>
+</figure>
 
 <hr class="command-separator">
 
-#### Delete a Member : `deleteMember`
+#### Edit a Member: `editMember`
+If you accidentally make a typo while creating your member, do not worry! You can quickly fix it with the `editMember` command.
 
-Deletes the member at the specified index.
+Here is how you can use it:<br>
+`editMember MEMBER_INDEX [n/MEMBER_NAME] [g/GENDER] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
 
-Format: `deleteMember MEMBER_INDEX`
+<div markdown="block" class="alert alert-info">
+:information_source: At least one field must be provided.
```

Comment:
Should we add something at the end to make it sound friendlier?
```suggestion
:information_source: At least one field must be provided for you to run the command.
```
or
```suggestion
:information_source: At least one field must be provided for the command to work.
```
not exactly sure if it's better.

Reply:
```suggestion
:information_source: Remember to provide at least one field to be edited!
```
How about this?

Reply:
@dylkaw's suggestion sounds good